storage/store.go

```diff
@@ -50,7 +50,7 @@ const (
 	GCResponseCacheExpiration = 1 * time.Hour
 	// raftIDAllocCount is the number of Raft IDs to allocate per allocation.
 	raftIDAllocCount                = 10
-	defaultRaftTickInterval         = 10 * time.Millisecond
+	defaultRaftTickInterval         = 100 * time.Millisecond
```

Comment:
add a TODO somewhere (or open an issue) to benchmark Raft with low tick interval to find bottlenecks. One such test should check that a tick doesn't hit the harddrive (at least not without any chance of it not being cached, such as in the current situation where it might be scanning over the uncommitted part of the log on ticks)